python/lsst/ip/isr/isrStatistics.py

```diff
@@ -317,15 +348,164 @@ def run(self, inputExp, ptc=None, overscanResults=None, **kwargs):
                      },
         )
 
-    def measureCti(self, inputExp, overscans, gains):
+    def measureCti(self, inputExp, untrimmedInputExp, gains):
+        """Task to measure CTI statistics.
+
+        Parameters
+        ----------
+        inputExp : `lsst.afw.image.Exposure`
+            The exposure to measure.
```

Comment:
Can you add a comment here that the full exposure is needed because the data region has the defect mask?